articles/azure-maps/release-notes-drawing-tools-module.md

```diff
@@ -14,6 +14,16 @@ ms.subservice: web-sdk
 
 This document contains information about new features and other changes to the Azure Maps Drawing Tools Module.
 
+## [2.0.0] (CDN: December 15, 2025, npm: TBD)
+
+### Changes
```

Comment:
Can we use `Breaking changes`? This is the same headings as [REST SDK release notes](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/maps/Azure.Maps.Search/CHANGELOG.md).